www/src/components/navigation-mobile.js

```diff
@@ -13,24 +12,22 @@ import presets, { colors } from "../utils/presets"
 import { svgStyles } from "../utils/styles"
 import typography, { rhythm, scale, options } from "../utils/typography"
 
-const getNavItemStyles = ({ isPartiallyCurrent }) =>
-  isPartiallyCurrent
-    ? {
-        className: css({
-          ...styles.link.default,
-          ...styles.link.active,
-          ...styles.svg.active,
-        }),
-      }
-    : {
-        className: css({
-          ...styles.link.default,
-          ...styles.svg.default,
-        }),
-      }
+const getProps = ({ isPartiallyCurrent }) =>
+  isPartiallyCurrent && { "data-active": true }
```

Comment:
How would you change it? (I don't like it that much either!)

Reply:
```
const getProps = ({ isPartiallyCurrent }) => {
  if (!isPartiallyCurren) {
    return;
  }
  return { "data-active": true }
}
```
it's more code but the minifier takes care of it.

Reply:
@wardpeet Why start with the falsy value in your if rather than the opposite? It saves 1 character by removing the `!`.

```
const getProps = ({ isPartiallyCurrent }) => {
  if (isPartiallyCurrent ) {
    return { "data-active": true }
  }
  return
}
```

Reply:
It's just the code style I use. It's called `Return Early`. I like to put meaningless returns like error state or null returns at the top and actual code that I should care about in the happy path at the bottom.

Let's not go into the world of micro-benchmarking it's most of the time not worth it. In this example, there will be nothing to win as a minifier just fixes this.

your snippet:
```js
const getProps = ({ isPartiallyCurrent }) => {
  if (isPartiallyCurrent) {
    return { "data-active": true }
  }
  return
}
```
minification:
```js
const getProps=({isPartiallyCurrent:t})=>{if(t)return{"data-active":!0}};
```

my snippet:
```js
const getProps = ({ isPartiallyCurrent }) => {
  if (!isPartiallyCurrent ) {
    return
  }
  return { "data-active": true }
}
```
minification:
```js
const getProps=({isPartiallyCurrent:t})=>{if(t)return{"data-active":!0}};
```

Reply:
@wardpeet No no, don't worry, I was simply asking out of curiosity. Thanks for the clarification. 😄